src/main/scala/beam/agentsim/agents/ridehail/RideHailManager.scala

```diff
@@ -1191,6 +1244,8 @@ class RideHailManager(
     val inquiryWithUpdatedLoc = RideHailRequest.handleImpression(inquiry, beamServices)
     rideHailResourceAllocationManager.respondToInquiry(inquiryWithUpdatedLoc) match {
       case NoVehiclesAvailable =>
+        beamServices.simMetricCollector
+          .writeIteration("ride-hail-inquiry-not-available", SimulationTime(inquiry.departAt))
```

Comment:
Should all these strings as the name of metric be extracted to some constants or enums?

Reply:
well .. because there are places where metric name is `calculated` during execution, it will be too complicated to implement it in proper way.
so, i suggest to leave it as is.